src/draw/shapes.typ

```diff
@@ -313,19 +313,18 @@
     )
 
     if mark_.check-mark(style.mark) {
-      let (marks, segments) = mark_.place-marks-along-path(ctx, style.mark, drawables.segments)
+      let (marks, segments) = mark_.place-marks-along-path(ctx, style.mark, transform, drawables)
       drawables.segments = segments
       drawables = (drawables,) +  marks
```

Comment:
If drawables are being passed completely, why not add the marks to the drawables and return only the drawables?
```
drawables = mark_.place-marks-along-path(ctx, style.mark, transform, drawables)
```